sparse/tests/test_core.py

```diff
@@ -136,6 +137,10 @@ def test_dot():
     assert_eq(a.dot(b), sa.dot(sb))
     assert_eq(np.dot(a, b), sparse.dot(sa, sb))
 
+    if sys.version_info >= (3, 5):
+        assert_eq(eval("a @ b"), eval("sa @ sb"))
+        assert_eq(eval("sa @ sb"), sparse.dot(sa, sb))
```

Comment:
Actually... no.  Your suggestion will simply exercise the `__matmul__` method of NumPy arrays, not the `__rmatmul__` method of sparse.  Let me fix that and push the change.